yammbs/torsion/_store.py

```diff
@@ -302,37 +310,79 @@ def optimize_mm(
                     ),
                 )
 
-    def get_log_sse(
+    def get_rmsd(
+        self,
+        force_field: str,
+        molecule_ids: list[int] | None = None,
+        skip_check: bool = False,
+    ) -> RMSDCollection:
+        """Get the RMSD summed over the torsion profile."""
+        from openff.toolkit import Molecule
+
+        if not molecule_ids:
+            molecule_ids = self.get_molecule_ids()
+
+        if not skip_check:
+            self.optimize_mm(force_field=force_field)
+
+        rmsds = RMSDCollection()
+
+        for molecule_id in molecule_ids:
+            if molecule_id not in molecule_ids:
+                continue
```

Comment:
Am I missing something magical here (or farther down the loop)? From the loop itself, it looks like this should never be entered.

Reply:
Whoops, that was pretty sloppy